docs/application/tizen-studio/release-notes/3-6-release-notes.md

```diff
@@ -0,0 +1,58 @@
+# Tizen Studio 3.6 Release Notes
+
+- Release Date: Dec 12, 2019
```

Comment:
Question. I've heard that the release date is today, Dec 10. 
Why the date is changed to Dec 12? Is there any different plan?

Reply:
thanks, had updated the wrong date by mistake